config/signatures.txt

```diff
@@ -0,0 +1,5 @@
+java.lang.Integer#parseInt(java.lang.String) @ Use oshi.util.ParseUtil#parseLastInt(java.lang.String,int)
```

Comment:
Just sticking this in `config` without mentioning it's for forbidden APIs is confusing.  I'd suggest either including "forbidden" in the file name or putting this file in a "forbidden" subdirectory.

Also all three of these should probably use the `parseFooOrDefault()` as the proposed suggestion (or at least the primary one) rather than the "last" one.

Reply:
Done